packages/core/src/rules/oas3/component-name-unique.ts

```diff
@@ -46,32 +49,47 @@ export const ComponentNameUnique: Oas3Rule | Oas2Rule = (options) => {
           const resolvedRef = resolve(ref);
           if (!resolvedRef.location) return;
 
-          addComponentFromAbsoluteLocation(
-            typeName,
-            resolvedRef.location.absolutePointer.toString()
-          );
+          addComponentFromAbsoluteLocation(typeName, resolvedRef.location);
         }
       },
     },
     Root: {
       leave(root: Oas3Definition, ctx: UserContext) {
         components.forEach((value, key, _) => {
-          if (value.size > 1) {
+          if (value.absolutePointers.size > 1) {
             const component = getComponentFromKey(key);
             const optionComponentName = getOptionComponentNameForTypeName(component.typeName);
-            const definitions = Array.from(value)
-              .map((v) => `- ${v}`)
-              .join('\n');
-
+            const definitions = Array.from(value.absolutePointers).join(', ');
```

Comment:
I'm not sure about this change, because it reformats the layout to be more difficult to read.

Previously we had

```
Component 'schemas/SomeSchema' is not unique. It is defined at:
- .../foobar.yaml#/components/schemas/SomeSchema
- .../test.yaml#/components/schemas/SomeSchema
```

Whereas now those all flow on one line, making it harder to read and compare the paths. The additional location information is great, but can we retain the original formatting on the output message please?

Reply:
Reverted it now to how it was before. You are right that it is more readable in the old format.